migrations/perf_copy_variant_test.go

```diff
@@ -0,0 +1,185 @@
+package migrations
+
+import (
+	"context"
+	"path/filepath"
+	"testing"
+
+	"github.com/evergreen-ci/evergreen"
+	evgdb "github.com/evergreen-ci/evergreen/db"
+	"github.com/evergreen-ci/evergreen/mock"
+	"github.com/evergreen-ci/evergreen/model"
+	"github.com/evergreen-ci/evergreen/testutil"
+	"github.com/mongodb/anser"
+	"github.com/mongodb/anser/db"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+	"gopkg.in/mgo.v2/bson"
+)
+
+type perfCopyVariantMigration struct {
+	migrationSuite
+}
+
+func TestPerfCopyVariantMigration(t *testing.T) {
```

Comment:
This function can be stripped down to just s := ... and suite.Run... The rest is handled by the migrationSuite

Reply:
done